guetzli/butteraugli_comparator.cc

```diff
@@ -31,6 +31,7 @@ ButteraugliComparator::ButteraugliComparator(const int width, const int height,
     : width_(width),
       height_(height),
       target_distance_(target_distance),
+      rgb_orig_(rgb),
```

Comment:
Please make the constructor parameter a const pointer if we now expect it be live as long as this class is live.